### PR TITLE
Update phoenix to 1.4.9 and change logout button to a regular link

### PR DESCRIPTION
**Pull request by aleciavogel**
This started out with me stumbling across #54 and #49. I simply wanted to eliminate the logout form and use the `link` helper method with the `method: :delete` attribute, but apparently that doesn't work because the version of Phoenix for this project was so out of date.

So, down the rabbit hole I went.

I only had credentials for Google at my disposal, so if someone can double-check that the other authentication methods remain intact, that would be lovely.

Hope this helps!

### CODE REVIEW

File: lib/ueberauth_example_web/controllers/auth_controller.ex [hunk 1]
```diff
@@ -13,6 +15,7 @@ defmodule UeberauthExampleWeb.AuthController do
   end
 
   def delete(conn, _params) do
+    Logger.debug("I ran")
```
Comment (yordis) [inline lib/ueberauth_example_web/controllers/auth_controller.ex]: Revert this

**Comment by yordis** [comment]
Once CI is passing I will test it locally.

Thank you for the work, appreciated.

File: lib/ueberauth_example_web/controllers/auth_controller.ex (hunk 1 above)
Comment (aleciavogel) [reply lib/ueberauth_example_web/controllers/auth_controller.ex]: whoopsie daisy, good catch

**Comment by Hanspagh** [comment]
Any news here or can we merge?

**Comment by doomspork** [comment]
@yordis any final thoughts on this?

**Pull request merged**